### PR TITLE
Theeran branch work

### PR DESCRIPTION
**Pull request by theeran-ak**

### CODE REVIEW

File: photo_upload.py [hunk 1]
```diff
@@ -76,9 +76,9 @@ def link_upload(link):
     if(request.method=="POST"):
         link=decrypt(link)
         os.remove(temp_photo_upload)
-        conn21=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline photo_upload.py:79]: old

File: photo_upload.py [hunk 2]
```diff
@@ -76,9 +76,9 @@ def link_upload(link):
     if(request.method=="POST"):
         link=decrypt(link)
         os.remove(temp_photo_upload)
-        conn21=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
+        conn21=pymysql.connect(host="localhost",user="root",password="2theeran7",database="woolyweb")
         cur21=conn21.cursor()
-        conn201=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline photo_upload.py:81]: old

File: Bank_details.py [hunk 3]
```diff
@@ -18,7 +18,7 @@ def submit_bank():
 def complete_bank(a):
     if (request.method=="POST"):
         lst=[]
-        obj2012=pymysql.connect(host="localhost",user="root",database="woolyweb",password="abcd")
```
Comment (K-Okid) [inline Bank_details.py:21]: old

File: Customer_company.py [hunk 4]
```diff
@@ -24,9 +24,9 @@ def submit_customer():
 def complete_customer(a):
     if(request.method=="POST"):
         r_customer.clear()
-        obj2=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline Customer_company.py:27]: old

File: Customer_company.py [hunk 5]
```diff
@@ -24,9 +24,9 @@ def submit_customer():
 def complete_customer(a):
     if(request.method=="POST"):
         r_customer.clear()
-        obj2=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
+        obj2=pymysql.connect(host="localhost",user="root",password="2theeran7",database="woolyweb")
         cur2=obj2.cursor()
-        obje=pymysql.connect(host="localhost",user="root",password="abcd",database="cust_company")
```
Comment (K-Okid) [inline Customer_company.py:29]: old

File: Farmers_check.py [hunk 6]
```diff
@@ -20,7 +20,7 @@ def a_submit_farmer_check():
 @app.route("/complete_farmer_check/<a>",methods=["POST","GET"])
 def complete_farmer_check(a):
     if(request.method=="POST"):
-        obj10=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline Farmers_check.py:23]: old

File: Farmers_personal.py [hunk 7]
```diff
@@ -23,10 +23,10 @@ def submit_farmer():
 def complete_farmer(a):
     if(request.method=="POST"):
         r_farmer.clear()
-        obj3=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline Farmers_personal.py:26]: old

File: Farmers_personal.py [hunk 8]
```diff
@@ -23,10 +23,10 @@ def submit_farmer():
 def complete_farmer(a):
     if(request.method=="POST"):
         r_farmer.clear()
-        obj3=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
+        obj3=pymysql.connect(host="localhost",user="root",password="2theeran7",database="woolyweb")
         cur3=obj3.cursor()
         cur4=obj3.cursor()
-        obj4=pymysql.connect(host="localhost",user="root",password="abcd",database="aadhar")
```
Comment (K-Okid) [inline Farmers_personal.py:29]: old

File: General_customer.py [hunk 9]
```diff
@@ -29,15 +29,15 @@ def complete_general(a):
         a=decrypt(a)
         a=json.loads(a)
         r_general.clear()
-        obj6=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline General_customer.py:32]: old

File: General_customer.py [hunk 10]
```diff
@@ -29,15 +29,15 @@ def complete_general(a):
...
+        obj6=pymysql.connect(host="localhost",user="root",password="2theeran7",database="woolyweb")
         cur6=obj6.cursor()
         cur7=obj6.cursor()
         cur7.execute("select aadhar from general_customer;")
         cur8=obj6.cursor()
         cur8.execute("select username from general_customer;")
         check3=list(cur8.fetchall())
         check2=list(cur7.fetchall())
-        objz=pymysql.connect(host="localhost",user="root",password="abcd",database="aadhar")
```
Comment (K-Okid) [inline General_customer.py:40]: old

File: Marketplace.py [hunk 11]
```diff
@@ -13,7 +13,7 @@ def marketplace():
 @app.route('/get_data_market')
 def get_data_market():
     farmers_marketplace.clear()
-    obj24 = pymysql.connect(host="localhost", user="root", password="abcd", database="woolyweb")
```
Comment (K-Okid) [inline Marketplace.py:16]: old

File: Product_description.py [hunk 12]
```diff
@@ -13,7 +13,7 @@ def description_cost():
     if request.method == "POST":
         description_list=[]
         description_list.clear()
-        obj222=pymysql.connect(host="localhost",user="root",database="woolyweb",password="abcd")
```
Comment (K-Okid) [inline Product_description.py:16]: old

File: Product_description.py [hunk 13]
```diff
@@ -24,7 +24,7 @@ def description_cost():
             return "<h1>PLEASE ENTER A VALID COST</h1><br><h1>THE COST SHOULD ONLY CONTAIN NUMBERS</h1>"
         cur222.execute("Insert into description values"+str(tuple(description_list))+"")
         obj222.commit()
-        obj223=pymysql.connect(host="localhost",user="root",database="woolyweb",password="abcd")
```
Comment (K-Okid) [inline Product_description.py:27]: old

File: Servers.py [hunk 14]
```diff
@@ -24,7 +24,7 @@ def submit_server():
 @app.route("/complete_server/<a>",methods=["POST","GET"])
 def complete_server(a):
     if(request.method=="POST"):
-        obj8=pymysql.connect(host="localhost",user="root",password="abcd",database="servers")
```
Comment (K-Okid) [inline Servers.py:27]: old

File: cust_company_check.py [hunk 15]
```diff
@@ -1,6 +1,6 @@
 import pymysql
 from en_de import*
-obj12=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline cust_company_check.py:3]: old

File: cust_company_check.py [hunk 16]
```diff
@@ -29,7 +29,7 @@ def complete_cust_company_check(a):
 
     if (request.method=="POST"):
         r_cust_company_check.clear()
-        obj12=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline cust_company_check.py:32]: old

File: general_customer_check.py [hunk 17]
```diff
@@ -24,7 +24,7 @@ def a_submit_generalcust_check():
 def complete_generalcust_check(a):
     if(request.method=="POST"):
         r_generalcust_check.clear()
-        obj11=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline general_customer_check.py:27]: old

File: server_authentication.py [hunk 18]
```diff
@@ -4,7 +4,7 @@
 import pymysql
 
 list_min_max=[0,0]
-obj13=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (K-Okid) [inline server_authentication.py:7]: old

File: server_authentication.py [hunk 19]
```diff
@@ -18,7 +18,7 @@ def server_authentication():
 def get_data():
     print(list_min_max)
     farmers.clear()
-    obj13=pymysql.connect(host="localhost",user="root",password="abcd",database="woolyweb")
```
Comment (theeran-ak) [inline server_authentication.py:21]: old

**Pull request merged**